hoomd/md/compute.py

```diff
@@ -282,84 +283,72 @@ def num_particles(self):
             return None
 
 
-class thermoHMA(Compute):
+class ThermoHMA(Compute):
```

Comment:
Should we name this explicitly `HarmonicallyMappedAverageThermodyanmicQuantities`? @b-butler what do you think?